view_presentation/interface/attribute_content_interface.py

```diff
@@ -0,0 +1,73 @@
+import pandas as pd
+from interface import interface
```

Comment:
interface is a class and should be capitalized as per PEP 8 (https://peps.python.org/pep-0008/#class-names). We should include a linter to avoid inconsistencies across different contributions. Until then, this is good to keep in mind to ease readability for whoever needs to read this later.